packages/web-components/src/components/cbp-button/cbp-button.tsx

```diff
@@ -100,13 +103,16 @@ export class CbpButton {
       }
     }
 
+    //console.log('this.button',this.button);
```

Comment:
I assume this would be nice for testing later but wanted to make sure you meant to leave it

Reply:
yeah, need to do more testing on slotted buttons